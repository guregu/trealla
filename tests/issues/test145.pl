:-initialization(main).

main :-
	Z = ([[a|Z],X]=[X,[Y|Z]]),
	Z,
	writeln(Z).

