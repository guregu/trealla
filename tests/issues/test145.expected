[[a|[[a|...],[a|[[a|...],_4]=[_4,[a|...]]]]=[[a|[[a|...],_4]=[_4,[a|...]]],[a|...]]],[a|[[a|...],_4]=[_4,[a|...]]]]=[[a|[[a|...],_4]=[_4,[a|...]]],[a|...]]
